<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DialogIVcurve</class>
 <widget class="QDialog" name="DialogIVcurve">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>898</width>
    <height>654</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QTabWidget" name="tabWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>881</width>
     <height>641</height>
    </rect>
   </property>
   <property name="currentIndex">
    <number>0</number>
   </property>
   <widget class="QWidget" name="tab">
    <attribute name="title">
     <string>BIAS Sweep</string>
    </attribute>
    <widget class="QCustomPlot" name="widgetIVgraph" native="true">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>861</width>
       <height>471</height>
      </rect>
     </property>
    </widget>
    <widget class="QGroupBox" name="groupBox">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>490</y>
       <width>861</width>
       <height>111</height>
      </rect>
     </property>
     <property name="title">
      <string>Bias Voltage Sweep Limits</string>
     </property>
     <widget class="QSpinBox" name="spinBoxBiasLow">
      <property name="geometry">
       <rect>
        <x>92</x>
        <y>35</y>
        <width>71</width>
        <height>29</height>
       </rect>
      </property>
      <property name="maximum">
       <number>4094</number>
      </property>
     </widget>
     <widget class="QSpinBox" name="spinBoxBiasStep">
      <property name="geometry">
       <rect>
        <x>221</x>
        <y>35</y>
        <width>71</width>
        <height>29</height>
       </rect>
      </property>
      <property name="minimum">
       <number>1</number>
      </property>
      <property name="maximum">
       <number>1000</number>
      </property>
     </widget>
     <widget class="QSpinBox" name="spinBoxBiasUpper">
      <property name="geometry">
       <rect>
        <x>397</x>
        <y>35</y>
        <width>71</width>
        <height>29</height>
       </rect>
      </property>
      <property name="minimum">
       <number>2</number>
      </property>
      <property name="maximum">
       <number>4096</number>
      </property>
     </widget>
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>40</y>
        <width>81</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>Lower limit:</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_2">
      <property name="geometry">
       <rect>
        <x>180</x>
        <y>40</y>
        <width>41</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>Step: </string>
      </property>
     </widget>
     <widget class="QLabel" name="label_3">
      <property name="geometry">
       <rect>
        <x>310</x>
        <y>40</y>
        <width>81</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>Upper limit:</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButtonStart">
      <property name="geometry">
       <rect>
        <x>555</x>
        <y>35</y>
        <width>84</width>
        <height>28</height>
       </rect>
      </property>
      <property name="text">
       <string>Start</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButtonStop">
      <property name="geometry">
       <rect>
        <x>735</x>
        <y>35</y>
        <width>84</width>
        <height>28</height>
       </rect>
      </property>
      <property name="text">
       <string>Stop</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButtonPause">
      <property name="geometry">
       <rect>
        <x>645</x>
        <y>35</y>
        <width>84</width>
        <height>28</height>
       </rect>
      </property>
      <property name="text">
       <string>Pause</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButtonSaveData">
      <property name="geometry">
       <rect>
        <x>735</x>
        <y>72</y>
        <width>84</width>
        <height>28</height>
       </rect>
      </property>
      <property name="text">
       <string>Save Data</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_4">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>76</y>
        <width>81</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>Channel: </string>
      </property>
     </widget>
     <widget class="QSpinBox" name="spinBoxChannelNumber">
      <property name="geometry">
       <rect>
        <x>92</x>
        <y>70</y>
        <width>71</width>
        <height>29</height>
       </rect>
      </property>
      <property name="maximum">
       <number>39</number>
      </property>
     </widget>
     <widget class="QLabel" name="label_5">
      <property name="geometry">
       <rect>
        <x>180</x>
        <y>75</y>
        <width>81</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>Upper VBIAS:</string>
      </property>
     </widget>
     <widget class="QDoubleSpinBox" name="spinBoxUpperBiasVoltage">
      <property name="geometry">
       <rect>
        <x>270</x>
        <y>70</y>
        <width>71</width>
        <height>29</height>
       </rect>
      </property>
     </widget>
    </widget>
   </widget>
   <widget class="QWidget" name="tab_2">
    <attribute name="title">
     <string>Breakdown Calculation</string>
    </attribute>
    <widget class="QCustomPlot" name="widgetBreakdownCalculation" native="true">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>861</width>
       <height>471</height>
      </rect>
     </property>
    </widget>
    <widget class="QGroupBox" name="groupBox_2">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>490</y>
       <width>861</width>
       <height>111</height>
      </rect>
     </property>
     <property name="title">
      <string>Breakdown Voltage Calculation</string>
     </property>
     <widget class="QPushButton" name="pushButtonSelectFile">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>30</y>
        <width>101</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>Select File</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_6">
      <property name="geometry">
       <rect>
        <x>120</x>
        <y>31</y>
        <width>31</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>File:</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEditSelectedFile">
      <property name="geometry">
       <rect>
        <x>150</x>
        <y>30</y>
        <width>691</width>
        <height>22</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButtonCalculate">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>60</y>
        <width>101</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>Calculate</string>
      </property>
     </widget>
     <widget class="QSpinBox" name="spinBoxXMin">
      <property name="geometry">
       <rect>
        <x>303</x>
        <y>60</y>
        <width>62</width>
        <height>23</height>
       </rect>
      </property>
      <property name="maximum">
       <number>1000</number>
      </property>
     </widget>
     <widget class="QSpinBox" name="spinBoxXMax">
      <property name="geometry">
       <rect>
        <x>303</x>
        <y>85</y>
        <width>62</width>
        <height>23</height>
       </rect>
      </property>
      <property name="maximum">
       <number>1000</number>
      </property>
     </widget>
     <widget class="QLabel" name="label_7">
      <property name="geometry">
       <rect>
        <x>147</x>
        <y>61</y>
        <width>161</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>Vbias filt. discard at front:</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_9">
      <property name="geometry">
       <rect>
        <x>149</x>
        <y>84</y>
        <width>151</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>Vbias filt. discard at back:</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_8">
      <property name="geometry">
       <rect>
        <x>374</x>
        <y>61</y>
        <width>151</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>Vbr calc. discard at front:</string>
      </property>
     </widget>
     <widget class="QSpinBox" name="spinBoxXGraphMax">
      <property name="geometry">
       <rect>
        <x>532</x>
        <y>85</y>
        <width>62</width>
        <height>23</height>
       </rect>
      </property>
      <property name="maximum">
       <number>1000</number>
      </property>
     </widget>
     <widget class="QSpinBox" name="spinBoxXGraphMin">
      <property name="geometry">
       <rect>
        <x>532</x>
        <y>60</y>
        <width>62</width>
        <height>23</height>
       </rect>
      </property>
      <property name="maximum">
       <number>1000</number>
      </property>
     </widget>
     <widget class="QLabel" name="label_10">
      <property name="geometry">
       <rect>
        <x>376</x>
        <y>84</y>
        <width>151</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>Vbr calc. discard at back:</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_11">
      <property name="geometry">
       <rect>
        <x>720</x>
        <y>61</y>
        <width>58</width>
        <height>15</height>
       </rect>
      </property>
      <property name="text">
       <string>Result:</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEditVbrResult">
      <property name="geometry">
       <rect>
        <x>770</x>
        <y>60</y>
        <width>71</width>
        <height>22</height>
       </rect>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QCustomPlot</class>
   <extends>QWidget</extends>
   <header>qcustomplot.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
